scripts/linux/publishReleasePackages.sh

```diff
@@ -239,7 +235,11 @@ publish_to_github()
 
         #Create Release Page
         url="https://api.github.com/repos/Azure/azure-iotedge/releases"
-        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" '{tag_name: $version, name: $version, target_commitish:"main", body: $body}')
+        reqBody='{tag_name: $version, name: $version, target_commitish:"main", draft: true, body: $body}'
+        if [[ $SKIP_UPLOAD == "false" ]]; then
+            reqBody='{tag_name: $version, name: $version, target_commitish:"main", body: $body}'
+        fi
+        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" "$reqBody")
```

Comment:
Can you explain what this line is doing? I'm having trouble understanding it.

Reply:
Bash-magic. It basically replacing the string `$version` of `reqBody` variable with the actual version string (i.e. 1.2.10).

Reply:
Recommend either adding a comment or splitting it up. Something to help with clarity.